source/common/config/config_provider_impl.h

```diff
@@ -216,10 +216,10 @@ class ConfigSubscriptionCommonBase
    * returns a updated/new version Config.
    * @param complete_cb the callback to run when the update propagation is done.
    */
+  using ConfigUpdateCb =
```

Comment:
Please make this top-level and commented, rather than part of the function signature.

Reply:
Still need to do this one :)

Reply:
Done. 
Sorry I missed this one earlier.